Packages/src/TypeScriptServer~/src/utils/content-length-framer.ts

```
@@ -0,0 +1,371 @@
+   * Creates a Content-Length framed message from JSON content.
+   * @param jsonContent The JSON content to frame
+   * @returns The framed message with Content-Length header
+   */
+  static createFrame(jsonContent: string): string {
+    if (!jsonContent) {
+      throw new Error(ContentLengthFramer.ERROR_MESSAGE_JSON_EMPTY);
+    }
+
+    const contentLength = Buffer.byteLength(jsonContent, ContentLengthFramer.ENCODING_UTF8);
+
+    if (contentLength > this.MAX_MESSAGE_SIZE) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)